src/languages/vbnet.js

```diff
@@ -33,30 +53,18 @@ export default function(hljs) {
     illegal: '//|{|}|endif|gosub|variant|wend|^\\$ ', /* reserved deprecated keywords */
     contains: [
       hljs.inherit(hljs.QUOTE_STRING_MODE, {contains: [{begin: '""'}]}),
-      hljs.COMMENT(
-        '\'',
-        '$',
-        {
-          returnBegin: true,
-          contains: [
-            {
-              className: 'doctag',
-              begin: '\'\'\'|<!--|-->',
-              contains: [hljs.PHRASAL_WORDS_MODE]
-            },
-            {
-              className: 'doctag',
-              begin: '</?', end: '>',
-              contains: [hljs.PHRASAL_WORDS_MODE]
-            }
-          ]
-        }
-      ),
+      commentMode,
       hljs.C_NUMBER_MODE,
       {
         className: 'meta',
-        begin: '#', end: '$',
-        keywords: {'meta-keyword': 'if else elseif end region externalsource'}
+        begin: '#(?:const|disable|else|elseif|enable|end|externalsource|if|region)\\b', end: '$',
```

Comment:
There are the only possible tags?

You don't need the `?:`, doesn't matter if we capture or not.

Reply:
Should we also be matching the start of a line here anywhere?  Or can these occur in the middle of a line?

Reply:
Removed matching group.

I think the statements can be indented, but not after other content.

Reply:
I dislike matching the whole line, but I say lets do it for specificity:

```
 begin: '^\s*#(const|disable|else|elseif|enable|en...
```

Ie, something like that... you could add a TODO comment to go back and hit this with `beforeMatch:` when that gets merged.